src/ts/helper/AdHelper.ts

```diff
@@ -76,7 +77,7 @@ export class AdHelper {
     return formattedErrorParts.join(' ');
   }
 
-  public static extractCsaiConvivaAdInfo(player: PlayerAPI, adBreakEvent: AdBreakEvent, adEvent: AdEvent): Conviva.ConvivaMetadata {
+  public static extractCsaiConvivaAdInfo(player: PlayerAPI, adBreakEvent: AdBreakEvent, duration: number, adEvent: AdEvent): Conviva.ConvivaMetadata {
```

Comment:
Similar to above, please rename the parameter to something like `mainCointentDuration`, otherwise it's quite confusing when someone reads this file/method.

Reply:
Renamed https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/118/commits/a0f340727fe1816302b51f74011dd2ef6d801ab6